.github/workflows/update-rds-ca-bundle.yml

```diff
@@ -0,0 +1,58 @@
+name: Update RDS CA Bundle
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '00 4 * * 1-5'
+
+permissions:
+  actions: write
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-rds-ca-inventory:
+    name: Update RDS CA Bundle
+    runs-on: pub-hk-ubuntu-24.04-ip
+    steps:
+      # - uses: actions/create-github-app-token@v1
+      #   id: generate-token
+      #   with:
+      #     app-id: ${{ vars.LINGUIST_GH_APP_ID }}
+      #     private-key: ${{ secrets.LINGUIST_GH_PRIVATE_KEY }}
+
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+
+      - name: Compare
+        id: compare
+        run: | 
+          wget -q https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem -O /tmp/global-bundle.pem
+          if cmp —silent — /tmp/global-bundle.pem heroku-24/rds-global-bundle.pem; then
```

Comment:
I chose simplicity over being exhaustive, I figure we don't add many new base images, so keeping this workflow updated isn't too hard